expirationd/init.lua

```diff
@@ -1311,6 +1310,9 @@ return {
     get_tasks      = get_tasks_obsolete,
     run_task       = run_task_obsolete,
     show_task_list = show_task_list_obsolete,
+
+    --- Current module version in string format `MAJOR.MINOR.PATCH`.
+    _VERSION = require('expirationd.version'),
```

Comment:
We need a unit-test for the public constant.

Reply:
Added unit test